packages/rum/src/boot/startRecording.spec.ts

```diff
@@ -173,6 +172,20 @@ describe('startRecording', () => {
     expect(secondSegment.records[0].type).toBe(RecordType.Meta)
   })
 
+  it('send view end record when the segment collection is waiting for the initial record', async () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could we clarify the intent of this test?

Reply:
What about: 'flushes pending mutations before ending the view, even after the segment has been flushed'?